tritium-caffeine/src/main/java/com/palantir/tritium/metrics/caffeine/CaffeineCacheMetricSet.java

```diff
@@ -95,6 +95,15 @@ protected Long loadValue() {
                     }
                 });
 
+        cache.policy().eviction().ifPresent(eviction -> cacheMetrics.put(
+                cacheMetricName("maximum", "size"),
+                new CachedGauge<Long>(clock, 500, TimeUnit.MILLISECONDS) {
+                    @Override
+                    protected Long loadValue() {
+                        return eviction.getMaximum();
+                    }
+                }));
```

Comment:
do we want to report weighted size as well?

```
        cache.policy().eviction().ifPresent(eviction -> {
            eviction.weightedSize().ifPresent(weightedSize -> cacheMetrics.put(
                    cacheMetricName("weighted", "size"),
                    (Gauge<Long>) () -> weightedSize));
            cacheMetrics.put(
                    cacheMetricName("maximum", "size"),
                    new CachedGauge<Long>(clock, 500, TimeUnit.MILLISECONDS) {
                        @Override
                        protected Long loadValue() {
                            return eviction.getMaximum();
                        }
                    });
        });
```

Reply:
I don't need it now, but imagined it could be useful in the future. Done.